app/scripts/views/mixins/password-strength-mixin.js

```diff
@@ -82,6 +84,7 @@ define(function (require, exports, module) {
             if (passwordCheckStatus === 'BLOOMFILTER_HIT' ||
```

Comment:
Was running through this and noticed that `password` and `12345678` does not trigger a warning. Dug into the password checker and there is a state `ALL_LETTERS_OR_NUMBERS` that I think should probably be accounted for here? Thoughts?